src/components/keypad/pre-algebra-page.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Keys from "../../data/key-configs";
+import {
+    KeypadPageContainer,
+    KeypadButton,
+    SecondaryKeypadButton,
+    KeypadActionButton,
+    PlaceHolderButton,
+} from "./keypad-page-items";
+
+import type {KeyConfig} from "../../data/key-configs";
+
+type Props = {|
+    onClickKey: (keyConfig: string) => void,
+|};
+type State = {};
+export default class NumericInputPage extends React.Component<Props, State> {
```

Comment:
super nit: we usually leave blank lines between these.  Also, `State` can be dropped because it's empty.